src/index.js

```diff
@@ -46,4 +26,26 @@ const getMyCityW = () => {
   return myCityDeserialized;
 };
 
+
+function displayData(inf) {
+  const obj = new Weather(inf.name, inf.weather[0].main, inf.weather[0].description,
+    inf.weather[0].icon, inf.main.temp);
+  obj.displayDomWeather();
+  localStorage.clear();
+}
+
+function showByCity() {
+  const url = getMyCityW();
+  if (url) {
+    fetch(url, { mode: 'cors' })
+      .then((response) => response.json())
```

Comment:
why is there another fetch here? i thought there is an api call file?

Reply:
let me explain you how my application work :
Actually, I tried to simulate the operation of openweathermap that shows the weather by detecting the location of the user (url = `${api}lat=${lat}&lon=${long}${myKey}`, or by using the location entered that why i used two fetch url = `${api}+city+${myKey}`.

Reply:
@zinagithub  Send me a slack message.